.phan/Shopware/Models/Attribute/Customer.php

```diff
@@ -5,4 +5,4 @@
 class Customer
```

Comment:
Are these still needed?

Reply:
It is in Components/Model/Customer.php, line 143

Reply:
👍